generators/app/templates/python/README.md

```diff
@@ -1,29 +1,65 @@
 # <%= name %> modular service
 
-*<%= name %>* is a Viam modular service that provides <%= api_name_lower %> capabilities
+This module implements the [<%= api_namespace %> <%= api_name_lower %> API](https://github.com/<%= api_namespace %>/<%= api_name_lower %>-api) in a <%= model %> model.
+With this model, you can...
 
-The model this module makes available is <%= model %>
+## Requirements
 
-## Prerequisites
-
-Add instructions here for any prerequisites.
+Add instructions here for any requirements.
 
 ``` bash
 ```
 
-## API
+## Build and Run
+
+To use this module, follow these instructions to [add a module from the Viam Registry](https://docs.viam.com/registry/configure/#add-a-modular-resource-from-the-viam-registry) and select the `<%= api_namespace %>:<%= api_name_lower %>:<%= model %>` model from the [`<%= model %>` module](https://app.viam.com/module/<%= api_namespace %>/<%= model %>).
 
-The <%= name %> resource implements the [<%= api_namespace %> <%= api_name_lower %> API](https://github.com/<%= api_namespace %>/<%= api_name_lower %>-api).
+## Configure your <%= api_name_lower %>
 
-Please also use the API codebase to interact with a configured version of this service via Viam SDK.
+> [!NOTE]  
+> Before configuring your <%= api_name_lower %>, you must [create a robot](https://docs.viam.com/manage/fleet/robots/#add-a-new-robot).
 
-## Viam Service Configuration
+Navigate to the **Config** tab of your robot’s page in [the Viam app](https://app.viam.com/).
+Click on the **Components** subtab and click **Create component**.
+Select the `<%= api_name_lower %>` type, then select the `<%= model %>` model. 
+Enter a name for your <%= api_name_lower %> and click **Create**.
 
-Add notes and example JSON config here.
+On the new component panel, copy and paste the following attribute template into your <%= api_name_lower %>’s **Attributes** box:
 
 ```json
+{
+  <INSERT SAMPLE ATTRIBUTES>
```

Comment:
I think the angular brackets are fine for placeholder values, since this is an entire blog I think this might look better?
```suggestion
  TODO: INSERT SAMPLE ATTRIBUTES
```

Reply:
Sounds good